pages/index.vue

```diff
@@ -83,7 +83,7 @@ p {
 }
 
 .hero-image {
-  background-image: url('/olivia-bliss-HTnycx0h_kk-unsplash.jpg');
```

Comment:
Also can you remove the other image if we're gonna stop using it?

Reply:
definitely :) done!